src/content/cloud/container-resources.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Container Resources
+description: Optimizing Container Resource Allocation for Efficient Development Environments
+sidebar_label: Container Resources
+id: container-resources
+---
+
+import Image from '@theme/Image';
+
+Okteto leverages the concept of Kubernetes resources to decide on which node to place each container.
+When you specify a resource `limit` for a container, Okteto enforces those limits so that the running container is not allowed to use more of that resource than the limit you set.
+Okteto also reserves at least the `request` amount of that system resources specifically for that container to use.
+Follow [this link](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) to know more about how Kubernetes resources work.
+
+## Best Practices
+
+Okteto monitors the CPU and Memory usage of your development environments to balance containers across all the nodes in the cluster.
+However, it's not capable of evicting containers when a node becomes overloaded.
+To avoid this situation, it's strongly recommended that you configure your CPU and Memory `requests` to align with your container's actual resource utilization.
+By doing so, you ensure that your containers always have sufficient resources available for optimal development speed.
+
+## Resource Consumption Warnings
+
+The Okteto UI shows warnings to assist you in setting the right resources for your development environments.
+If a development environment is utilizing over 50% of its CPU or Memory requests, you'll receive a yellow warning:
+
+<p align="center"><Image src="/docs/cloud/images/container-resources-yellow.png" alt="container resources yellow warning" width="650" /></p>
+
+If it's using more than 100% of its CPU and Memory requests, you'll receive a red warning:
+
+<p align="center"><Image src="/docs/cloud/images/container-resources-red.png" alt="container resources red warning" width="650" /></p>
+
+
+## Access your Metrics with kubectl
+
+If you require more detailed data on your container resource consumption, you can use the `kubectl top pods` command.
```

Comment:
@pchico83 This is off-topic, but does this command become obsolete when we eventually add our own data infrastructure for customers to consume information exactly like this?

Reply:
Not sure. This is accessible by developers. I think the developer insights feature is more targeted to admins